### DISCUSSION TITLE
An option to disable filter text input

### DISCUSSION THREAD

**Discussion by vkrms** [orchidjs/tom-select#379] [General]
I need to disable the filter text input for the design I'm currently implementing. If I hide the input with css, the dropdown begins to act funny and closes when the first item is selected. Apparently because the plugin has to focus on the now unfocusable input.

**Comment by oyejorge** [comment]
> If I hide the input with css, the dropdown begins to act funny and closes when the first item is selected. 

Do you have an example (jsfiddle or other) to demonstrate the problem?

**Reply by vkrms** [reply]
I'll make one

**Reply by vkrms** [reply]
https://jsfiddle.net/hntsevmo/1/

**Reply by oyejorge** [reply]
I see. [Here's what I would probably do](https://jsfiddle.net/ew46jm12/).
This uses css that make the input hidden but still accessible and a little js to reset the input when items are added.

**Reply by vkrms** [reply]
Thanks a lot! That helps for now, but will you please consider adding the option to disable the input at the initialization?
